pb/dns.proto

```diff
@@ -9,4 +9,41 @@ message DnsPacket {
 
 service DnsService {
 	rpc Query (DnsPacket) returns (DnsPacket);
+	rpc Watch (stream WatchRequest) returns (stream WatchResponse);
+}
+
+message WatchRequest {
```

Comment:
We should add a Makefile to convert to protos to go code

Reply:
Done in a separate PR